torch/_torch_docs.py

```diff
@@ -3952,23 +3954,27 @@ def merge_dicts(*dicts):
     tensor([ True,  True, False, False])
 """.format(**common_args))
 
-# TODO: see https://github.com/pytorch/pytorch/issues/43667
-add_docstr(torch.logspace,
-           """
-logspace(start, end, steps=100, base=10.0, *, \
+# TODO: update kwargs formatting (see https://github.com/pytorch/pytorch/issues/43667)
+add_docstr(torch.logspace, """
+logspace(start, end, steps, base=10.0, *, \
          out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
 """ + r"""
-Returns a one-dimensional tensor of :attr:`steps` points
-logarithmically spaced with base :attr:`base` between
+
+Creates a one-dimensional tensor of size :attr:`steps` whose values interpolate
+logarithmically from :attr:`start` to :attr:`end`, inclusive.
```

Comment:
This would be better with an "example" of the form (start_value, 2nd_value, ..., end_value)

Update language to "interpolate with a logarithmic scale", maybe "exponents of the logarithm are linearly interpolated"